config/config.default.php

```diff
@@ -338,13 +338,13 @@
     // Available T-Shirt sizes, set value to null if not available
     'tshirt_sizes'            => [
```

Comment:
Those changes should be accompanied with a migration that changes the values of the `users_personal_data.shirt_size` column.